Platform/AlderlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -458,9 +458,13 @@ BoardInit (
     if (FeaturePcdGet (PcdEnablePciePm)) {
       PciePmConfig ();
     }
-    Status = SetFrameBufferWriteCombining (0, MAX_UINT32);
-    if (EFI_ERROR(Status)) {
-      DEBUG ((DEBUG_INFO, "Failed to set GFX framebuffer as WC\n"));
+    // UEFI Payload will change cache type to UC based on PCI root bridge
```

Comment:
Is this specific to RPL-P or can this solution be safely applied to all ADL as well?

Reply:
The hang has only been seen on RPL-P so far. However, on all ADL the UEFI payload will undo the WriteCombining setting.

Reply:
Okay, thanks.